codex-rs/core/src/command_safety/is_safe_command.rs

```diff
@@ -2,6 +2,14 @@ use crate::bash::try_parse_bash;
 use crate::bash::try_parse_word_only_commands_sequence;
 
 pub fn is_known_safe_command(command: &[String]) -> bool {
+    #[cfg(target_os = "windows")]
+    {
+        use super::windows_safe_commands::is_safe_command_windows;
+        if is_safe_command_windows(command) {
+            return true;
+        }
+    }
+
```

Comment:
This is making me question whether we should be doing `is_safe_to_call_with_exec()` as it works today on Windows. Possibly only if you have Git Bash enabled or whatever it is on Windows and we're actually going to run the command through it? What do you think?